docs/roadmap.md

```diff
@@ -1,38 +1,120 @@
+<!-- markdownlint-disable MD024 -->
+
 # Roadmap
 
-The aim is to deliver a Minimum Viable Product (MVP). In a nutshell, an MVP
-seeks to collect the maximum amount of validated learning about our users in a
-short time.
+## 2022
+
+### Quarter 1
+
+- Brainstorm process
+
+---
+
+### Quarter 2
+
+- Initial project setup
+- First tests with RGW
+- [UI mockups][1]
+
+---
+
+### Quarter 3
+
+- File-based backend
+- Basic S3 operations support
+- User management
+- Helm charts
+- Testing
+- UI implementation
+- Documentation
+- Basic object management & versioning
+- Bucket management
+- Automation implementation
+
+---
+
+### Quarter 4
+
+- Object deletion/undeletion
+- Multipart uploads
+- Multipart copy
+- ACL support
+- UI: Basic object explorer
+- Rancher Partner chart
 
-Based on the user feedback we collect, we develop features that add up towards
-the goal of delivering the MVP. We're working on phases:
+## 2023
 
-## Phase 1
+### Plan of record
 
-- Use SQLite as storage backend.
-- Running the s3gw in a container.
-- Be able to run basic S3 operations.
-- Ability to deploy the S3 Gateway on a K8s or K3s cluster with.
-- [Design mockups][1] for User management & an S3 explorer.
+Features the team is committed to deliver within the timeframe.
+
+#### Quarter 1
+
+- [Expiration Lifecycle Management (unversioned)][2]
+- [Object locks][3]
+- [Object listing with prefixes/filters][4]
+- [Telemetry][5]
+- [UI: Object deletion/undeletion][7]
+
+---
+
+#### Quarter 2
+
+- [Expiration Lifecycle Management (versioned)][8]
+- [Access & Identity Management][10]
+- [Metrics from backend][9]
+- [UI: Object versioning][11]
 
 ---
 
-## Phase 2
+#### Quarter 3
 
-- Basic S3 explorer UI
-- Helm chart
-- Define & start work on a file-based backend
-- Implement CI/CD
+- [Implement Amazon Security Token][12]
 
 ---
 
-## Phase 3
+#### Quarter 4
 
-- File-based backend integration
-- Increased support for S3 functional features
-- S3 explorer UI
-- User management UI
+- [Metrics on UI][15]
 
 ---
 
-[1]: https://www.figma.com/file/IeozuvvYlrKBs7qm030dyo/S3-Wireframe?node-id=0%3A1
+### Plan of direction
+
+Features the team wants to develop but are currently not a top priority.
+
+- [Cross-site replication][13]
+- [Implement Storage classes][14]
+- [CRD based configuration of buckets and rbac][6]
+
+### In scope
+
+Features that aren't currently planned but the project would accept external
+contributions.
+
+- [SSO][16]
+
+### Out of scope
+
+Features that aren't being considered unless core assumptions change.
+
+!!! Info
+    Please note that this roadmap is an evergreen document and will most
+    certainly evolve as we continue to learn from our users.
+
```

Comment:
A divider here would be nice to indicate that the footnotes aren't the features we consider out of scope.